PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -891,14 +885,10 @@ export class ParaReportPublishService {
         return uuid.v5(violType + ruleId + msg + severity + lineHash + uri + order, namespace);
     }
 
-    private isStaticAnalysisReportNameDuplicate = (sourcePath: string): boolean => {
-        const filename = path.basename(sourcePath);
-        if (this.staticAnalysisReportsMap.has(filename)) {
-            tl.warning(`Skipping report with duplicate names: ${sourcePath}`);
-            return true;
-        }
-        this.staticAnalysisReportsMap.set(filename, sourcePath);
-        return false;
+    private renameSarifReport = (sourcePath: string): string => {
+        const fileName = path.basename(sourcePath);
+        const nFileName = sourcePath.replace(':\\', '_').replaceAll('/', '.').replaceAll('\\', '.');
```

Comment:
Please consider linux system, for example the path is '/path/to/report.xml', and the result is '.path.to.report.xml', this file will be hidden because it starts with '.'